internal/dwarf/frame/table.go

```diff
@@ -253,38 +258,49 @@ const low_6_offset = 0x3f
 
 type instruction func(ctx *Context)
 
-func NewContext() *Context {
+func NewContext(fullExecutablePath string) *Context {
```

Comment:
We don't need to pass this down, we just wrap the error returned by `BuildCompactUnwindTable` with `fmt.Errorf("build compact unwind table for executable %q: %w", fullExecutablePath, err)`